tests/resources/telescope_positions-North-utm.meta.yml

```diff
@@ -27,22 +27,23 @@ CTA:
   PRODUCT:
     DESCRIPTION: |-
       Telescope positions for the CTAO Northern site
-      (UTM coordinate system).
+      (UTM coordinate system). Consistent with
+      document CTA-SPE-PSC-000000-0001 1c.
```

Comment:
This includes the revision number (the `c` in `1c`). Do we want to keep that or just the document number and perhaps major version?

Reply:
Hmm, I don't see why we shouldn't have this. Remember that this is a test file only.

Reply:
OK, just wanted to make sure you noticed.